cookbook/core/Dockerfile

```diff
@@ -16,9 +16,11 @@ RUN pip3 install awscli
 # Install gcloud for GCP
 RUN apt-get install curl --assume-yes
 
-RUN curl -sSL https://sdk.cloud.google.com | bash
-ENV PATH $PATH:/root/google-cloud-sdk/bin
-
+WORKDIR /opt
+RUN curl https://sdk.cloud.google.com > install.sh
```

Comment:
why not write directly to `/opt/install.sh`? That directory exists in the base image.